From 14a4eb9a3de5031bbcfdfd99e68c8c822e31e98b Mon Sep 17 00:00:00 2001
From: Joachim Wiberg <troglobit@gmail.com>
Date: Sat, 20 Jul 2024 10:45:19 +0200
Subject: [PATCH] Add extra linefeed to wall message to ensure it's seen.

Signed-off-by: Joachim Wiberg <troglobit@gmail.com>
---
 src/syslogd.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/syslogd.c b/src/syslogd.c
index e6d8747..f7a4ee5 100644
--- a/src/syslogd.c
+++ b/src/syslogd.c
@@ -2079,7 +2079,9 @@ void fprintlog_write(struct filed *f, struct iovec *iov, int iovcnt, int flags)
 		f->f_time = timer_now();
 		logit("\n");
 		pushiov(iov, iovcnt, "\r\n");
-		wallmsg(f, &iov[0], iovcnt);
+		/* Make sure it's seen, may be followed by reboot */
+		pushiov(iov, iovcnt, "\r\n");
+		wallmsg(f, iov, iovcnt);
 		break;
 	} /* switch */
 
