diff -Naur cmake-3.31.8.orig/Source/CTest/cmCTestCurl.h cmake-3.31.8/Source/CTest/cmCTestCurl.h
--- cmake-3.31.8.orig/Source/CTest/cmCTestCurl.h	2025-09-12 16:34:29.585631359 +0200
+++ cmake-3.31.8/Source/CTest/cmCTestCurl.h	2025-09-12 16:34:56.632659149 +0200
@@ -52,7 +52,7 @@
   std::vector<std::string> HttpHeaders;
   std::string HTTPProxyAuth;
   std::string HTTPProxy;
-  curl_proxytype HTTPProxyType;
+  long HTTPProxyType;
   bool UseHttp10 = false;
   bool Quiet = false;
   int TimeOutSeconds = 0;
