diff --git a/src/rejmerge.in b/src/rejmerge.in
index e875d09..e83af74 100755
--- a/src/rejmerge.in
+++ b/src/rejmerge.in
@@ -350,11 +350,7 @@ main() {
 	done
 
 	# Remove empty directories.
-	find "$_rejected_dir" -depth -type d | while read -r _dir; do
-		if [ "$_dir" != "$_rejected_dir" ]; then
-			rmdir "$_dir" >/dev/null 2>&1
-		fi
-	done
+	find "$_rejected_dir" -depth -mindepth 1 -empty -type d -delete >/dev/null 2>&1
 
 	if [ "$REJECTED_FILES_FOUND" = "no" ]; then
 		echo "Nothing to merge"
