diff --git a/src/rejmerge.in b/src/rejmerge.in
index 80047ec..a804dd3 100644
--- a/src/rejmerge.in
+++ b/src/rejmerge.in
@@ -489,11 +489,7 @@ main() {
 
 	# Remove empty directories after all files have been
 	# processed.
-	find "$_rejected_dir" -depth -type d | while read -r _dir; do
-		if [ "$_dir" != "$_rejected_dir" ]; then
-			rmdir "$_dir" >/dev/null 2>&1
-		fi
-	done
+	find "$_rejected_dir" -depth -mindepth 1 -empty -type d -delete >/dev/null 2>&1
 
 	# If no rejected files were found or processed, print a
 	# message.
